### PR TITLE
VIH-8174 Update start hearing endpoint in line with kinlys changes

### PR DESCRIPTION
**Pull request by robertbennett1998**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/VIH-8174

### Change description ###
- Updated start hearing endpoint to iterate all host roles and add them as participants to force transfer into the hearing. This should mimic the existing behaviour.

Dependent on https://github.com/hmcts/vh-video-api/pull/408 being merged.

**Does this PR introduce a breaking change?** (check one with "x")
```
[x] Yes
[ ] No
```

### CODE REVIEW

File: VideoWeb/VideoWeb.AcceptanceTests/VideoWeb.AcceptanceTests.csproj [hunk 1]
```diff
@@ -63,7 +63,7 @@
     <PackageReference Include="TestApi.Client" Version="1.27.0" />
     <PackageReference Include="UserApi.Client" Version="1.25.0" />
     <PackageReference Include="VH.AcceptanceTests.Common" Version="1.21.39" />
-    <PackageReference Include="VideoApi.Client" Version="1.27.6" />
+    <PackageReference Include="VideoApi.Client" Version="1.27.7-pullrequest0408-0002" />
```
Comment (omio-ma) [inline VideoWeb/VideoWeb.AcceptanceTests/VideoWeb.AcceptanceTests.csproj:66]: looks to be referring to a pull request version, dont think its intentional?

File: VideoWeb/VideoWeb.IntegrationTests/VideoWeb.IntegrationTests.csproj [hunk 2]
```diff
@@ -24,7 +24,7 @@
     <PackageReference Include="NUnit" Version="3.13.0" />
     <PackageReference Include="NUnit3TestAdapter" Version="3.17.0" />
     <PackageReference Include="UserApi.Client" Version="1.24.3" />
-    <PackageReference Include="VideoApi.Client" Version="1.27.6" />
+    <PackageReference Include="VideoApi.Client" Version="1.27.7-pullrequest0408-0002" />
```
Comment (omio-ma) [inline VideoWeb/VideoWeb.IntegrationTests/VideoWeb.IntegrationTests.csproj:27]: appears to be everything pretty sure this needs to be changed to a more stable version

Comment (robertbennett1998) [reply VideoWeb/VideoWeb.IntegrationTests/VideoWeb.IntegrationTests.csproj:27]: Need video api to be merged

**Review by kevin-hmcts — DISMISSED** [review]
LGTM

**Review by omio-ma — APPROVED** [review]

**Review by PrasannaDommalapati — APPROVED** [review]
LGTM

**Pull request merged**